README.md

```diff
@@ -36,21 +36,21 @@
 
 ## Installation
 
-### Option 1 - Using Files
+### Using Files
```

Comment:
Also these headings are kinda vague? Not sure exactly what to change them to but yeah thought I'd mention it!

Reply:
Thanks, looks good, reasonable changes.